lib/email_updater.rb

```diff
@@ -5,9 +5,10 @@ class EmailUpdater
 
   attr_reader :user
 
-  def initialize(guardian = nil, user = nil)
+  def initialize(guardian = nil, user = nil, initiating_user = nil)
```

Comment:
3 optional arguments might be the threshold were it's better to use optional _named_ arguments

```suggestion
  def initialize(guardian: nil, user: nil, initiating_user: nil)
```

Reply:
Damn I knew I should have done that!